Tests/FluentAssertions.Specs/Collections/GenericCollectionAssertionsSpecs.cs

```diff
@@ -362,6 +362,34 @@ public void When_single_item_matching_a_predicate_is_found_it_should_allow_conti
             act.Should().Throw<XunitException>().WithMessage("Expected*greater*4*2*");
         }
 
+        [Fact]
+        public void When_single_item_contains_brackets_it_should_not_fail()
```

Comment:
The method names are a bit inconsistent with the assertions.
We do want the assertions to fail, but not due to a `FormatException`.

```suggestion
        public void When_single_item_contains_brackets_it_should_format_them_properly()
```